CHANGELOG.txt

```diff
@@ -2755,8 +2726,7 @@ Improvement
 - [AIRFLOW-3591] Fix start date, end date, duration for rescheduled tasks (#4502)
 - [AIRFLOW-3709] Validate `allowed_states` for ExternalTaskSensor (#4536)
 - [AIRFLOW-3522] Add support for sending Slack attachments (#4332)
-- [AIRFLOW-3569] Add "Trigger DAG" button in DAG page (/www only) (#4373)
-- [AIRFLOW-3569] Add "Trigger DAG" button in DAG page (/www_rbac only) (#4373)
+- [AIRFLOW-3569] Add "Trigger DAG" button in DAG page (#4373)
```

Comment:
Was this one fixed manually?

Reply:
Yes. They all were. The script doesn't try to resolve the duplicate, it just fails if it finds one.